frontend/src/src-tauri/postinstall.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Function to update permissions for a given path pattern
+update_permissions() {
+  local pattern="$1"
+  local path=$(dpkg -L picto-py | grep "$pattern" | head -n 1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)